BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -376,6 +391,21 @@ public override void OnStart(StartState state)
         {
             //base.OnStart(state);
 
+            initialMass = part.mass;
+            if (boosterMass < 0 || boostTime <= 0)
+            {
+                if (BDArmorySettings.DEBUG_MISSILES && boosterMass < 0) Debug.LogWarning("[BDArmory.MissileBase]:Error Bad Configured Missile");
+                boosterMass = 0;
```

Comment:
You're going to want a ``if (boostermass > initialMass*0.8f) { boostermass = initialMass * 0.8f;}`` or ``boosterMass = MathF.Clamp(boostermass, initialMass*0.1f, initialMass *0.8f);`` check to prevent having a fuelmass greater than the missile mass.

Reply:
i did something similar, it checks the max fuel on total to be 90% of the missile mass, and do the clamps